logging/cluster-logging-upgrading.adoc

```diff
@@ -1,20 +1,31 @@
 :_content-type: ASSEMBLY
 :context: cluster-logging-upgrading
 [id="cluster-logging-upgrading"]
-= Updating OpenShift Logging
+= Updating Logging
 include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-[id="cluster-logging-supported-versions"]
-== Supported Versions
-For version compatibility and support information, see link:https://access.redhat.com/support/policy/updates/openshift#logging[Red Hat OpenShift Container Platform Life Cycle Policy]
+There are two types of {logging} updates: minor release updates (5.x.x) and major release updates (5.x).
```

Comment:
nit: Do you think it makes sense aligning the versioning we use internally to customers as well instead of just calling it `x`
```suggestion
There are two types of {logging} updates: minor release updates (5.y.z) and major release updates (5.y).
```

Reply:
I like this much better